.github/workflows/public_docker_images.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       matrix:
         # Must match version at https://www.python.org/ftp/python/
         python: ["3.9.18", "3.10.13", "3.11.8"]
-        framework: ["tf", "torch"]
+        framework: ["tf", "torch", "tf-extras", "torch-extras"]
```

Comment:
@frgfm @odulcy-mindee Not sure do we want this as "prod" "non prod" (with -extras) !? :sweat_smile: